modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -206,3 +206,22 @@
 .question-seen {
 	font-weight: 400;
 }
+
+.questions-root .breadcrumb {
+	.breadcrumb-item {
+		font-size: 1.15rem;
+	}
+
+	.breadcrumb-item + .breadcrumb-item {
+		padding-left: 0.7rem;
+	}
+	.questions-breadcrumb-unstyled {
+		padding: 0 0.3rem;
+	}
+	.questions-breadcrumb-unstyled:hover {
+		background-color: rgba(39, 40, 51, 0.06);
```

Comment:
Mmm... is it not available in clay?

Reply:
Emiliano typically adds a comment with the name of the color in clay

Reply:
There is no comment for the background color in this case and is not the `$secondary` color

Reply:
Maybe we can use a variable of clay? https://clayui.com/docs/css/color.html#secondary